independent-publisher-connectors/Microsoft Learn Catalog/apiDefinition.swagger.json

```diff
@@ -0,0 +1,780 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Learn Catalog",
+    "description": "A connector to the Microsoft Learn Catalog API",
```

Comment:
@SeanKe11y please remove the word "API" from the description. Please follow our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector) and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info).

Reply:
Removed